services-manager/README.md

```diff
@@ -21,14 +21,16 @@ The source for this file can be found at `services-manager/src/resources/doc/REA
 
 # <img src="../logos/logo.svg" width="180">
 
-## A Tool for Complex and Scalable Data Access Policy Enforcement
+### Palisade Services Manager
```

Comment:
Would have expected the line A Tool for Complex...  to be there.

Reply:
Added this back in